src/stream-tcp-reassemble.c

```diff
@@ -1107,9 +1107,9 @@ static bool GetAppBuffer(const TcpStream *stream, const uint8_t **data, uint32_t
                     "got data at %"PRIu64". GAP of size %"PRIu64,
                     offset, blk->offset, blk->offset - offset);
             *data = NULL;
-            *data_len = blk->offset - offset;
+            *data_len = (uint32_t)(blk->offset - offset);
```

Comment:
not actually sure if we can't get a gap > 4GiB... It seems unlikely for sure. I guess we'd get all confused anyway due to sequence number wrapping around.

Reply:
How could we tell that the GAP is over 4Gbytes with TCP ?

> I guess we'd get all confused anyway due to sequence number wrapping around.

Indeed